metaflow/plugins/kfp/kfp.py

```diff
@@ -194,8 +194,10 @@ def _command(
         def copy_log_cmd(log_file):
             cp_command = environment.get_boto3_copy_command(
                 s3_path=(
-                    f"{{datastore_root}}/{self.flow.name}/{{run_id}}/{step_name}"
-                    f"/${TASK_ID_ENV_NAME}/{log_file}"
+                    os.path.join(
+                        "{datastore_root}",
+                        f"/{self.flow.name}/{{run_id}}/{step_name}/${TASK_ID_ENV_NAME}/{log_file}",
```

Comment:
We can remove the first `/` in this line now.

Reply:
`os.path.join` will handle that.  But it's true it can be removed -> it's a style change.